lib/establish_connection.py

```diff
@@ -81,6 +81,22 @@ def get_input_params():
         default=False,
         dest='deprecate_rest',
         help='/api and /rest rendering - with /rest being deprecated')
+    parser.add_argument(
+        '-fam',
+        '--fetch-authentication-metadata',
+        required=False,
+        nargs='?',
+        const=True,
+        default=False,
+        dest='fetch_auth_metadata',
+        help='retrieves authentication information from the authentication metadata service')
+    parser.add_argument(
+        '-ars'
+        '--auto-rest-services',
+        nargs='+',
+        default=[],
+        dest='auto_rest_services',
+        help='list of services described with auto rest')
```

Comment:
Could you please elaborate a bit more here (having this small sentence leaves the taste of low quality, unusable, buggy software).

Please explain that this is the "List of REST API services available in vCenter 6.0 (which got Automatic REST rendering)." And why would one wnat to specify it (for prformance gains).